EShop.Web/Views/Products/Index.cshtml

```diff
@@ -19,6 +19,8 @@
                     <h5 class="card-title">@product.Name</h5>
                     <p class="card-text">@product.Description</p>
                 </div>
+                <a class="btn btn-danger mt-1" asp-controller="Products" asp-action="DeleteProduct" asp-route-id="@product.Id"> Delete </a>
+                <a class="btn btn-outline-success mt-1" asp-controller="Products" asp-action="EditProduct" asp-route-id="@product.Id"> Edit Product </a>
             </div>
         </div>
     }
```

Comment:
As avea o cerinta un pic mai complexa, vreau sa modifici comportamentul de delete, cand dai delete:

Te va duce pe o pagina de confirmare cu un mesaj:

`Are you sure you want to delete @Model.Name`?`

Si un buton de confirmare. Dupa stergere, redirect inapoi la lista.

Reply:
De asemenea https://getbootstrap.com/docs/4.0/components/card/ te rog sa folosesti bootstrap 4 sa stilizezi cardul cu continut si butoane sa arete frumos 😄 (aici ma refer la lista)